fw/http.c

```diff
@@ -7425,15 +7425,59 @@ __cfgop_brange_hndl(TfwCfgSpec *cs, TfwCfgEntry *ce, unsigned char *a)
 	return 0;
 }
 
-#define TFW_HTTP_CFG_CUSTOM_BRANGE(name)				\
+static int
+__cfgop_brange_parse_disallowed(char *str, unsigned char *a)
+{
+	char *start = str, *end = str;
+
+	while (true) {
+		if (*end == ' ' || *end == '\0') {
```

Comment:
```suggestion
		if (*end != ' ' && *end != '\0')
		        continue;
```
The line width limit in the coding style mostly targets smaller nesting in the code